django/contrib/admin/helpers.py

```diff
@@ -235,6 +236,17 @@ def __init__(self, inline, formset, fieldsets, prepopulated_fields=None,
         if prepopulated_fields is None:
             prepopulated_fields = {}
         self.prepopulated_fields = prepopulated_fields
+        verbose_name = self.opts.verbose_name
+        if verbose_name:
+            verbose_name = verbose_name[0].upper() + verbose_name[1:]
+        self.inline_formset_data = json.dumps({
+            'name': "#%s" % self.formset.prefix,
+            'options': {
+                'prefix': self.formset.prefix,
+                'addText': force_text(_('Add another %(verbose_name)s') % {'verbose_name': verbose_name}),
+                'deleteText': force_text(_('Remove'))
```

Comment:
Instead of using `_` which is module defined alias for `django.utils.translation.ugettext_lazy` and force the conversion of the returned promise to `six.text_type` I suggest you use `django.utils.translation.ugettext` directly.